Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp

```diff
@@ -294,15 +310,24 @@ namespace AZ::Render
             }
 
             m_characterPhysicsDebugDraw.RenderRagdollColliders(
-                                debugDisplay, ragdollColliderConfiguration, nodeDebugDrawDataFunction, parentIndices,
-                                { renderActorSettings.m_ragdollColliderColor, renderActorSettings.m_selectedRagdollColliderColor,
-                                  renderActorSettings.m_violatedRagdollColliderColor });
+                debugDisplay,
+                ragdollColliderConfiguration,
+                nodeDebugDrawDataFunction,
+                parentIndices,
+                { renderActorSettings.m_ragdollColliderColor,
```

Comment:
suggestion: Same comment about adding type for object created here just before the braces

Reply:
Done